client/components/QuestionsAndAnswers/QAndAComponents/Helpful.jsx

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { makeStyles, Grid, Typography } from '@material-ui/core';
+import axios from 'axios';
 
 const useStyles = makeStyles((theme) => ({
   underlined: {
     textDecoration: 'underline'
   }
 }));
 
-const Helpful = ( {helpfulness} ) => {
+const Helpful = ({ helpfulness }) => {
   const classes = useStyles();
-  const handleHelpfulClick = () => {
-    console.log('clicked helpful');
+  const [helpful, setHelpful] = useState(helpfulness);
+  const [clicked, setClicked] = useState(false);
+
+  const handleClick = () => {
+    if (!clicked) {
+      setHelpful(helpfulness + 1);
+      setClicked(true);
+    }
   }
+
   return (
     <div>
       <Typography variant="caption">Helpful?   </Typography>
-      <Typography variant="caption" onClick={handleHelpfulClick} className={classes.underlined}>Yes</Typography>
-      <Typography variant="caption">  ({helpfulness})</Typography>
+      <Typography variant="caption" onClick={handleClick} className={classes.underlined}>Yes</Typography>
```

Comment:
Future styling suggestion: change from cursor to pointer when hovering over 'Yes'